docs/pages/versions/unversioned/sdk/stripe.md

```diff
@@ -66,6 +66,10 @@ For usage information and detailed documentation, please refer to:
 If you're relying on redirects, you'll need to pass in a `urlScheme` to `initStripe`. To make sure you always use the proper `urlScheme`, pass in:
 
 ```js
+import Constants, { ExecutionEnvironment } from 'expo-constants';
```

Comment:
should we include the `Linking` import as well?

Reply:
Nice point! Will do it now.

Reply:
```suggestion
import * as Linking from 'expo-linking';
import Constants, { ExecutionEnvironment } from 'expo-constants';
```